generic/utils/ras.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import commands
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class RASTools(Test):
+
+    def setUp(self):
+        Arch = commands.getoutput("arch")
+        if "ppc" not in Arch:
+            self.skip("supported only on Power platform")
+        sm = SoftwareManager()
+        if not sm.check_installed("ppc64-diag") and not sm.install("ppc64-diag"):
+            self.error('ppc64-diag is needed for the test to be run')
+        if not sm.check_installed("powerpc-utils") and not sm.install("powerpc-utils"):
+            self.error('powerpc-utils is needed for the test to be run')
+        if not sm.check_installed("lsvpd") and not sm.install("lsvpd"):
+            self.error('lsvpd is needed for the test to be run')
+
+    def test(self):
+        scriptToRun = self.params.get("script", default=None)
+        toolToRun = self.params.get("tool", default=None)
+        dir = os.path.join(self.datadir, 'log')
+        if not os.path.exists(dir):
+            os.makedirs(dir)
+        commandToRun = './ras.py.data/' + scriptToRun + ' ' + \
+            '>>' + ' ' + 'ras.py.data/log/output.txt' + ' ' + '2>&1'
+        var = os.system(commandToRun)
+        print "Return code is", var
+        cmd_result = process.run(commandToRun, ignore_status=True)
+        self.log.info(cmd_result)
+        if (var == 0):
```

Comment:
here you should be checking `cmd_result.exit_status`. Also it's duplicate to log the `0` return code, because the exit code already logged by `process.run`.